internal/scorecard/helpers/test_definitions.go

```diff
@@ -116,6 +116,13 @@ func (ts *TestSuite) ApplySelector(selector labels.Selector) {
 	}
 }
 
+// List prints out tests and removes the test from being executed
+func (ts *TestSuite) List() {
+	for i := 0; i < len(ts.Tests); i++ {
+		fmt.Printf("%s\n", ts.Tests[i].GetName())
+	}
```

Comment:
Shows that are missing the checks to ensure that all worked as expected.

Reply:
that function is removed in this update.